datadog_checks_dev/datadog_checks/dev/tooling/commands/release.py

```diff
@@ -817,6 +817,46 @@ def changelog(ctx, check, version, old_version, initial, quiet, dry_run):
         write_file(changelog_path, changelog_buffer.getvalue())
 
 
+def build_wheel(package_path, sdist):
```

Comment:
Let's put this in https://github.com/DataDog/integrations-core/blob/master/datadog_checks_dev/datadog_checks/dev/tooling/release.py